WORKSPACE

```diff
@@ -78,3 +78,9 @@ git_repository(
     commit = "3d73b4fecbd18de77588ab5eef712d50f34f601e",
     remote = "https://github.com/GoogleCloudPlatform/runtimes-common.git",
 )
+
+http_file(
+    name = "ubuntu_tar_download",
+    url = "http://cdimage.ubuntu.com/ubuntu-base/releases/16.04/release/ubuntu-base-16.04-core-amd64.tar.gz",
```

Comment:
I think this location is better: https://partner-images.canonical.com/core/xenial/current/

Reply:
done